source/sorted_table_model.ts

```diff
@@ -77,7 +77,30 @@ export class SortedTableModel extends TableModel {
     } else {
       this.order = [];
```

Comment:
By default, fields should have the same name as the parameter, ie. ```this.columnOrder```.

Reply:
Typescript doesn't like that because of the get columnOrder method.

Reply:
I'm not sure what I should call it. Or should underscore the name?